starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2879,10 +2918,13 @@ sub set_ccd_temps {
           sprintf("CCD temperature exceeds %.1f C\n", $config{ccd_temp_red_limit});
     }
 
-    # Add CRITICAL if OR and too cold as fid lights may be out of boxes
-    if (($self->{obsid} < 38000) and ($self->{ccd_temp_acq} < -14.0)) {
+# Add CRITICAL if OR and temperature looks well out of range (fid drift model may be wrong)
```

Comment:
I was considering this more a sanity check on the fid drift model inputs -- with the removal of the -14C check we have no automated check on "starcheck's estimate is too cold" and we rarely look at the model plot in general.  But if you don't see any value there that's fine -- I'll caution it for now and plan to remove it when I've finished the check with a comparison to the proseco temperature values.

Reply:
> more a sanity check on the fid drift model inputs

For a sanity check on the starcheck CCD temperature predicts vs. proseco then this is the wrong place and has an inappropriate conditional (must occur during an OR) and then gives a confusing warning message (fid positions are a red herring since the the root cause problem is about a bad CCD temperature propagation).

Reply:
Fair. I suppose I'll worry less about wrong kadi data going into thermal predicts once kadi state validation is 100% live, which may be before this PR is approved!